docs/pages/blog/mui-x-v7-beta.md

```diff
@@ -0,0 +1,247 @@
+
+### Date Object support in filter model
+
+The `filterModel` now supports `Date` objects for `date` and `dateTime` column types, providing a more intuitive and efficient filtering experience.
+
+While string values remain compatible for these types, any updates to the `filterModel` made through the UI (such as via the filter panel) will now automatically use `Date` objects, ensuring consistency and ease of data handling.
+
+### New stable features
+
+During major versions, MUI X releases new features under the `experimental` flag as a failsafe in case there's a need to change the API based on user feedback.
+
+We're now excited to announce that the following features have been promoted to stable and don't require the use of the flag anymore.
+
+- [Cell selection](https://next.mui.com/x/react-data-grid/cell-selection/) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+- [Clipboard pasting](https://next.mui.com/x/react-data-grid/clipboard/#clipboard-paste) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan 'Premium plan')
+- [Column grouping](https://next.mui.com/x/react-data-grid/column-groups/)
+
+## TreeView
+
+Following the promotion of the component from the labs to MUI X, our primary focus has been on improving its developer experience.
+This includes efforts to clarify documentation and improve key examples, making them more informative and user-friendly.
+
+A particularly significant initiative is the time-saving alternative to the traditional JSX architecture, the RichTreeView.
+
+### RichTreeView
+
+We've evolved the TreeView into two distinct components.
+
+The first is the `SimpleTreeView`, which retains the classic and familiar JSX approach and is still the one behind the original `<TreeView />` tag.
+
+The second is the `RichTreeView` , a new variant designed to streamline the development process, and with which we aim to support several common use cases out-of-the-box.
+
+Similar to the DataGrid, it requires only a dataset and possibly a few properties to render; it does all the heavy lifting for you, handling most of the complexities internally.
+
+```jsx
+const myDataSet = [
+    {
+        id: 'node-1',
+        label: 'Node 1',
+        children: [
+            { id: 'node-1-1', label: Node 1.1' },
+            { id: 'node-1-2', label: Node 1.2' },
+        ],
+    },
+        {
+        id: 'node-2',
+        label: 'Node 2',
+    },
+];
+```
+
+```jsx
+<RichTreeView
+  items={myDataSet}
+  defaultCollapseIcon={<ExpandMoreIcon />}
+  defaultExpandIcon={<ChevronRightIcon />}
+/>
+```
+
+Check out the [new component](https://next.mui.com/x/react-tree-view/rich-tree-view/items/)!
+
+## Charts
+
+Since the initial stable release of the charts components a few months ago, we have been diligently listening to your feedback and focusing on refining the foundational charts to ensure they effectively cater to your diverse use cases.
```

Comment:
I think it's best to treat this as its own mini-open source project

```suggestion
Since the initial stable release of MUI X Chart a few months ago, we have been diligently listening to your feedback and focusing on refining the foundational charts to ensure they effectively cater to your diverse use cases.
```